crates/amplifier-api/src/identity.rs

```diff
@@ -0,0 +1,138 @@
+//! Helpers for creating an authenticated `reqwest::Client` for amplifier-api.
```

Comment:
This code is moved from the previous implementation. It was at `crates/amplifier-api/src/client.rs`.